modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,174 @@
+[[yomi.installer]]
+= Install with Yomi
+
+Yomi (yet one more installer) is an installer for openSUSE operating systems.
+It is a SaltStack state, and can be used for installing Salt clients.
+
+Yomi consists of two components:
+
+* The Yomi formula, which contains the Salt states and modules required to drive the installation.
+For the formula, see https://build.opensuse.org/package/show/systemsmanagement:yomi/yomi-formula.
+* The LiveCD operating system image, which includes the `pre-configured ``salt-minion`` service.
+For the image, see https://build.opensuse.org/package/show/systemsmanagement:yomi/openSUSE-Tumbleweed-Yomi
```

Comment:
This is correct too. Also there is a pxe-yomi-image-opensuse15 package in Uyuni (or will be there soon). I will add installation instructions once the SR is created and accepted.

Reply:
I'll update when this changes.